### PR TITLE
LPS-196165 Add section label for fragment options

### PR DESCRIPTION
**Pull request by claraizquierdo**
Motivation
=======
Add title of the section in the fragment configuration for  fragments: Content Display, Content Flags and Content Ratings
[Link to story or ticket](https://liferay.atlassian.net/browse/LPS-196165)

Steps to Verify:
----------------
1. Go to page editor
1. Add fragments Content Display, Content Flags and Content Ratings
1. Check that in their configuration options there is a section with title "Fragment-Name Options"

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/ContentObjectFragmentRenderer.java [hunk 1]
```diff
@@ -62,6 +62,8 @@ public String getCollectionKey() {
 	public String getConfiguration(
 		FragmentRendererContext fragmentRendererContext) {
 
+		Locale locale = fragmentRendererContext.getLocale();
```
Comment (ealonso) [inline modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/ContentObjectFragmentRenderer.java:65]: @claraizquierdo it is better if you inline it

File: modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/ContentObjectFragmentRenderer.java [hunk 2]
```diff
@@ -77,7 +79,13 @@ public String getConfiguration(
 						).put(
 							"typeOptions",
 							JSONUtil.put("enableSelectTemplate", true)
-						))))
+						))
+				).put(
+					"label",
+					_language.format(
```
Comment (ealonso) [inline modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/ContentObjectFragmentRenderer.java:85]:
```
@claraizquierdo something like

() -> {
    Locale locale = fragmentRendererContext.getLocale();

    return _language.format(
						locale, "x-options",
						_language.get(locale, "content-ratings"));
}
```

**Comment by ealonso** [comment]
ci:forward

**Comment by rafaellajordao** [comment]
The failures are not related to this PR [(comparison with acceptance routine here)](https://testray.liferay.com/home/-/testray/runs/compare?compare=true&view=details&statusA=3&testrayRunIdB=2428672771&testrayRunIdA=2430401368&statusB=2). See https://github.com/brianchandotcom/liferay-portal/pull/141260

**Comment by rafaellajordao** [comment]
ci:close

**Pull request closed without merging**